### DISCUSSION TITLE
Thoughts on a Discord?

### DISCUSSION THREAD

**Discussion by Jonneal3** [mckaywrigley/chatbot-ui#1338] [Ideas]
I think instead of the GitHub forums for discussion related stuff, can we move this to a discord? Im happy to host it/start it or have a temporary one until @mckaywrigley makes one? Im sure its in the works but i'm in on this thing in real time to debug for some betas Ive got n my hosted version and I think real-time feedback by the community (or as close to that is possible) might be worth it while we all get into this thing...Thoughts?

**Comment by Quitoboss** [comment]
There will never be any problem with an "unofficial" discord existing since the author himself has not commented on the matter.
You can try it, I like the idea, however keep in mind that if there aren't many activity here either, it may not work either, or it may be what revitalizes the community...

**Comment by Jonneal3** [comment]
@Quitoboss Already got one - https://discord.gg/G9t6TCSpk4 -- Join here - Got like 80+ ppl in there now
